### PR TITLE
grpc ping support compression option

### PR DESCRIPTION
**Pull request by wulianglongrd**

### CODE REVIEW

**Comment by ldemailly** [comment]
Thanks (and sorry I only saw your MR accidentally today looking at dependanbot ones... don't hesitate to @ me)

Is compression on both client and server side, how is it negotiated? it seems in the MR you only change the client side

**Comment by wulianglongrd** [comment]
> Thanks (and sorry I only saw your MR accidentally today looking at dependanbot ones... don't hesitate to @ me)
> 
> Is compression on both client and server side, how is it negotiated? it seems in the MR you only change the client side

thanks @ldemailly . Yes, I have changed the server. The server side only needs to import "google.golang.org/grpc/encoding/gzip"

File: fgrpc/pingsrv.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	_ "google.golang.org/grpc/encoding/gzip" // Install the gzip compressor
```
Comment (wulianglongrd) [inline fgrpc/pingsrv.go:31]: hello @ldemailly , here is change the server side.  refer [here](https://github.com/grpc/grpc-go/blob/master/examples/features/compression/server/main.go#L30)

**Comment by wulianglongrd** [comment]
friendly ping @ldemailly

**Comment by wulianglongrd** [comment]
@ldemailly PTAL

**Comment by ldemailly** [comment]
I really need to fix github notifications, only seeing this now by manually going over open PRs

**Review by ldemailly — APPROVED** [review]

**Pull request merged**